ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/connected/relations/GeospatialRelationsAttributer.java

```diff
@@ -22,7 +22,7 @@ public TreeRelation getRelationToParentFragment(LdesFragment childFragment) {
 
 	private String getWKT(LdesFragment currentFragment) {
 		String fragmentWKT = currentFragment.getValueOfKey(FRAGMENT_KEY_TILE).orElseThrow(
-				() -> new MissingFragmentValueException(currentFragment.getFragmentId(), FRAGMENT_KEY_TILE));
+				() -> new MissingFragmentValueException(currentFragment.getFragmentId().asString(), FRAGMENT_KEY_TILE));
```

Comment:
Small suggestion: if we frequently need the fragment id string, it might be worthwhile to create a method on LdesFragmentIdentifier : `getFragmentIdString()`.

That avoids that you after a few more iterations have code such as someObject.someMethod().someMethod().someMethod().someMethod()

https://en.wikipedia.org/wiki/Law_of_Demeter